containers/AddFly.js

```diff
@@ -26,7 +30,7 @@ class AddFly extends Component {
         <FlyForm />
         
         <View style={{flex: 1, flexDirection: 'row', justifyContent: 'center'}}>
-          <TouchableOpacity style={styles.button}
+          <TouchableOpacity color='white' style={styles.button}
             onPress = {() => {
               this.props.clearFlyFields();
               this.props.navigation.navigate('MyFlyBox');
```

Comment:
Nice! Thank you for adding this in. This will clear the Fly Form entry once a user either cancel or submit a fly.